spec/spec_helper.rb

```diff
@@ -14,7 +14,7 @@
 
   def assert_correct_indenting(extension='rb', string)
     whitespace = string.scan(/^\s*/).first
-    string = string.split("\n").map { |line| line.gsub /^#{whitespace}/, '' }.join("\n").strip
+    string = string.split("\n").map { |line| line.gsub(/^#{whitespace}/, '') }.join("\n").strip
```

Comment:
seems weird to interpolate `whitespace` here. Why is the `scan` happening anyway? Why is it not just `line.gsub(/^\s*/, '')`?